WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsDeactivateStripeShopManagerView.swift

```diff
@@ -0,0 +1,69 @@
+import SwiftUI
+import Yosemite
+
+struct InPersonPaymentsDeactivateStripeShopManager: View {
```

Comment:
💭 it looks like the only difference between this view and `InPersonPaymentsDeactivateStripeAdmin` is the button to manage plugins? what do you think if we pass in a boolean that indicates whether to show this button with `viewModel.userIsAdministrator` in `InPersonPaymentsViewController`?